crates/astria-conductor/local.env.example

```diff
@@ -46,11 +46,19 @@ ASTRIA_CONDUCTOR_PRETTY_PRINT=false
 # is set to `true`.
 NO_COLOR=
 
-# The URL to a fully trusted CometBFT/Sequencer to issue cometbft RPCs. Example
-# RPCs are subscribing to new blocks, fetching blocks at a given level, or
-# retrieving validators.
+# The URL of the Sequencer gRPC service for requesting new blocks.
+# 127.0.0.1:8080 is the default socket address for its gRPC server.
+ASTRIA_CONDUCTOR_SEQUENCER_GRPC_URL="http://127.0.0.1:8080"
+
+# The URL of a fully trusted CometBFT/Sequencer node to get the latest height of
+# the Sequencer network, and to get the validators for a given Sequencer height
+# to verify Sequencer data read off Celestia.
 # 127.0.0.1:26657 is the default socket address in comebft's `rpc.laddr` setting.
-ASTRIA_CONDUCTOR_SEQUENCER_URL="ws://127.0.0.1:26657/websocket"
+ASTRIA_CONDUCTOR_SEQUENCER_COMETBFT_URL="http://127.0.0.1:26657"
+
+# The duration in milliseconds that sequencer-relayer sleeps before
+# requesting the latest block height from sequencer.
+ASTRIA_CONDUCTOR_SEQUENCER_BLOCK_TIME_MS=1000
```

Comment:
This is about half of the best case scendario block time is that a good ratio or should this be 2000?

Reply:
2000 is our default right? I have changed it to that number and have noted that that's the default.